clients/cloud/kotlin/README.adoc

```diff
@@ -0,0 +1,142 @@
+= Kotlin Example Confluent Cloud Client
+
+NOTE: Produce messages to and consume messages from https://www.confluent.io/confluent-cloud/[Confluent Cloud] using the Kotlin version of Java Producer and Consumer, and Kafka Streams API.
+
+== Prerequisites
+
+* Java 1.8 or higher to run the demo application
```

Comment:
Is there no Kotlin-specific pre-req?

Reply:
Not really. Kotlin standard library is in application class path